### ISSUE TITLE
App generator freezes on rnpm-link

### ISSUE THREAD

**Issue by khelil** [infinitered/ignite#54]
Hi Guy'z,

I have problems using app generator.
Running `ignite new MyApplication` freezes on rnpm-link command.

![capture d ecran 2016-04-14 a 09 41 12](https://cloud.githubusercontent.com/assets/28292/14520496/2186f1fc-0225-11e6-8ab3-4a6f2817a038.png)

All file seems to be there, i can run the application on ios, but i got an error on android :

```
Execution failed for task ':app:installDebug'.
> com.android.builder.testing.api.DeviceException: Timeout getting device list.
```

This error occured after updating rnpm. Before the update i was able to generate an application that runs fine on android but had a missing font error on iOS.

Any ideas ?
Thanks !

**Comment by khelil** [comment]
issue resolved after running `rnpm link` into repo !

**Comment by skellock** [comment]
Thank you @khelil .

I'm going to reopen this as I experienced this problem as well.

It only started after upgrading to `1.6.2` of `rnpm`.  

Relevent issue that was marked fixed as of `1.6.1` https://github.com/rnpm/rnpm/issues/138

Digging deeper reveals:

* https://github.com/SBoudrias/Inquirer.js/issues/356
* https://github.com/lelandrichardson/react-native-maps/issues/166

I'll verify version numbers and re-run tonight with & without ignite.

**Comment by skellock** [comment]
Looked a bit deeper tonight.  Can't seem to figure it out.  It is still hanging for me though.  :(

It seems to happen after the the `rnpm link` happens.  

I tried installing a fresh version of React Native and then manually installed installed `react-native-vector-icons`, `react-native-i18n`, and `react-native-push-notification`.  Then I ran `rnpm link` and everything worked great.

I'm out of ideas.

`rnpm` will be merging into `react-native-cli` shortly, so I guess we wait it out?

Sorry I couldn't help more.
